libraries/standard/http/CMakeLists.txt

```diff
@@ -7,8 +7,7 @@ add_library( http
 
 # HTTP public include path.
 target_include_directories( http PUBLIC ${HTTP_INCLUDE_PUBLIC_DIRS}
-                                        ${LOGGING_INCLUDE_DIRS}
-                                        ${CMAKE_CURRENT_LIST_DIR}/utest )
```

Comment:
This was initially a hack to include the config header when building http library but is no longer needed because the demo already has a config header included.

Reply:
Can the logging include directory dependency be added in the demos as well?

Reply:
Yes, `${LOGGING_INCLUDE_DIRS}` is already part.